src/pages/version/EnqueueModal.tsx

```diff
@@ -78,6 +78,9 @@ export const EnqueuePatchModal: React.FC<EnqueueProps> = ({
       data-cy="enqueue-modal"
     >
       <InputLabel htmlFor={COMMIT_MESSAGE_ID}>Commit Message</InputLabel>
+      <CommitSquashWarning>
+        Warning: submitting a patch to the commit queue will squash the commits.
+      </CommitSquashWarning>
       <StyledTextArea
         id={COMMIT_MESSAGE_ID}
         value={commitMessageValue}
```

Comment:
This is actually a good opportunity to replace this component with a [leafygreen text area component](https://www.mongodb.design/component/text-area/example/). It has a description attribute that would be perfect for this type of message. 


![image](https://user-images.githubusercontent.com/4605522/137993445-97da1080-a427-4822-8af1-8b033bde5dcc.png)

Reply:
Good idea 😅 Will change to leafygreen!